<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cont_det_form</class>
 <widget class="QDialog" name="cont_det_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>441</width>
     <height>137</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="varset_1">
    <item row="0" column="0">
     <widget class="QLabel" name="image_in_path_label">
      <property name="text">
       <string>image in path</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="image_in_ledit"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="image_in_browse_btn">
      <property name="text">
       <string>browse...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="image_out_path_label">
      <property name="text">
       <string>image out path</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="image_out_ledit"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="image_out_browse_btn">
      <property name="text">
       <string>browse...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>299</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="help_btn">
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="launch_test_btn">
      <property name="text">
       <string>Launch test</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
